chainforge/react-server/src/backend/utils.ts

```diff
@@ -19,7 +19,7 @@ const fetch = require('node-fetch');
 
 /** Where the ChainForge Flask server is being hosted, if any. */
 // @ts-ignore
-export const FLASK_BASE_URL = (window.__CF_HOSTNAME !== undefined && window.__CF_PORT !== undefined) ? `http://${window.__CF_HOSTNAME}:${window.__CF_PORT}/` : 'http://localhost:8000/';
+export const FLASK_BASE_URL = (window.__CF_HOSTNAME !== undefined && window.__CF_PORT !== undefined) ? '/' : 'http://localhost:8000/';
```

Comment:
this seems suspect.
it says, if you pass in a host or a port, use `/` else use `localhost`. // im not sure thats what you want

in my unsent pr,  i passed in a value to explicitly set flask_base_url via __CF_FLASK_BASE_URL 
```
export const FLASK_BASE_URL = window.__CF_FLASK_BASE_URL !== undefined ?  window.__CF_FLASK_BASE_URL | (
(window.__CF_HOSTNAME !== undefined && window.__CF_PORT !== undefined) ? `http://${window.__CF_HOSTNAME}:${window.__CF_PORT}/` : 'http://localhost:8000/');
```

not perfect code, but i think this shows you the concern.

Reply:
I realized that just the slash works. The localhost is for the situation where the app is not being served via Flask —which happens only during development, when we run on localhost:3000.